invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/controls/PublishButton/SubmitReviewModal.js

```diff
@@ -15,6 +15,13 @@ import { Button, Checkbox, Form, Icon, Message, Modal } from "semantic-ui-react"
 import * as Yup from "yup";
 
 export class SubmitReviewModal extends Component {
+  componentDidMount() {
+    // A11y: Focus the first input field in the form
+    const firstFormFieldWrap = document.getElementById("accept-access-checkbox");
+    const checkboxElem = firstFormFieldWrap.querySelector("input");
+    checkboxElem?.focus();
```

Comment:
I think using .ref is more "react way" of doing this, instead of using vanilla js, but this is not a strong opinion

Reply:
I had trouble accessing the element through ref which is why I did it like this, but I can give it another try if it's worth the time. Let me know.

Reply:
I would try just for the sake of consistency ;)